api/v1beta2/moduletemplate_types.go

```diff
@@ -78,6 +78,18 @@ type ModuleTemplateSpec struct {
 	// +kubebuilder:validation:MinLength:=3
 	Channel string `json:"channel"`
 
+	// Version identifies the version of the Module. It can be empty, or a semantic version.
```

Comment:
```suggestion
	// Version identifies the version of the Module. Can be empty, or a semantic version.
```

To be aligned with the description of ModuleName?